packages/modules/fsm/src/components/inbox/FSMLink.js

```diff
@@ -68,8 +67,13 @@ const FSMLink = ({ isMobile, data }) => {
       <div className="complaint-links-container">
         {GetLogo()}
         <div className="body">
-          {links.map(({ link, text, hyperlink = false, notAccessTo = [] }, index) => {
-            if (!notAccessTo?.includes(COLLECTOR)) {
+          {links.map(({ link, text, hyperlink = false, accessTo = [] }, index) => {
```

Comment:
no role based filtering applied as of now

Reply:
pls check below code